src/mongocxx/validation_criteria.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright 2015 MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <mongocxx/validation_criteria.hpp>
+
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/builder/stream/helpers.hpp>
+
+#include <mongocxx/config/prelude.hpp>
+
+namespace {
+
+///
+/// Convert validation levels to strings.
```

Comment:
I don't think you need to doc-comment unnamed namespace methods since users can't call them. Can comment them with // if you want.